src/lib.rs

```diff
@@ -98,6 +88,19 @@
 //! At this time, we do not recommend use for security-critical applications.
 //! Use at your own risk.
 //!
+//! # Useful features
+//!
+//! The main interfaces allow a [`Participant`] to process messages from
+//! other participants, producing a set of outgoing messages and, if the
+//! protocol completed, an output.
+//!
+//! The library includes logic to handle messages received outside the strict
+//! bounds of the round structure. When messages are received early, they are
+//! cached in memory by the library and handled at the appropriate time.
```

Comment:
I would change the wording on this last sentence to "and are retrieved + processed at the appropriate time"